### PR TITLE
Adds mobile tokens. [Amends #2085]

### PR DESCRIPTION
**Pull request by humancompanion-usds**

### CODE REVIEW

File: src/_data/tokens/vads-color-semantic.csv [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
+vads-color-action-surface-destructive-on-dark,Semantic,vads-color-action-surface-destructive-on-dark,uswds-system-color-red-vivid-40,#fb5a47,,Used for button on mobile app in dark mode
 vads-color-base,Semantic,vads-color-base,uswds-system-color-gray-90,#1b1b1b,,
 vads-color-base-dark,Semantic,vads-color-base-dark,uswds-system-color-gray-cool-60,#565c65,vads-color-gray,
 vads-color-base-darker,Semantic,vads-color-base-darker,uswds-system-color-gray-cool-70,#3d4551,vads-color-gray-dark,
 vads-color-base-darkest,Semantic,vads-color-base-darkest,uswds-system-color-gray-90,#1b1b1b,vads-color-base,Also vads-color-ink
 vads-color-base-light,Semantic,vads-color-base-light,uswds-system-color-gray-cool-30,#a9aeb1,vads-color-gray-light,
 vads-color-base-lighter,Semantic,vads-color-base-lighter,uswds-system-color-gray-cool-10,#dfe1e2,vads-color-gray-lighter,
 vads-color-base-lightest,Semantic,vads-color-base-lightest,uswds-system-color-gray-5,#f0f0f0,vads-color-gray-lightest,
+vads-color-bg-default-on-dark,Semantic,vads-color-bg-default-on-dark,uswds-system-color-gray-warm-90,#171716,,Used for app background on mobile app in dark mode
```
Comment (danbrady) [inline src/_data/tokens/vads-color-semantic.csv:14]:
```suggestion
vads-color-background-default-on-dark,Semantic,vads-color-background-default-on-dark,uswds-system-color-gray-warm-90,#171716,,Used for app background on mobile app in dark mode
```
All* of our tokens do not abbreviate "background" as "bg" but rather spell it out. My preference is to spell everything out so there's no guessing. However, I think the mobile team has adopted "fg" and "bg" already. 

<sup>*</sup>All except vads-color-inset-bg

**Review by danbrady — APPROVED** [review]

File: src/_data/tokens/vads-color-semantic.csv (hunk 1 above)
Comment (jamigibbs) [reply src/_data/tokens/vads-color-semantic.csv:14]: @danbrady Did they read your mind? https://github.com/department-of-veterans-affairs/component-library/pull/1157

Comment (humancompanion-usds) [reply src/_data/tokens/vads-color-semantic.csv:14]: I asked them to make the same change. Just wasn't sure if they would do it quickly. Fix forthcoming.

Comment (danbrady) [reply src/_data/tokens/vads-color-semantic.csv]: Whoa! 😮 

Maybe I was channelling @jessicawoodin when I was reviewing!

**Pull request merged**